src/Microsoft.DotNet.XHarness.TestRunners.Common/TcpTextWriter.cs

```diff
@@ -194,8 +194,9 @@ private static string SelectHostName(string[] names, int port)
 
             return result;
         }
-
+#if NET6_0
```

Comment:
@premun  is this correct?

Reply:
@mandel-macaque please have a look at https://github.com/dotnet/xharness/pull/607 and let's merge that first. It will remove the need for all of these changes you're making here